src/libstd/time/mod.rs

```diff
@@ -9,6 +9,14 @@
 // except according to those terms.
 
 //! Temporal quantification.
+//!
+//! Example:
+//!
+//! ```
+//! use std::time::Duration;
+//!
+//! let five_seconds = Duration::new(5, 0);
```

Comment:
I think [`Duration::from_secs()`](https://doc.rust-lang.org/nightly/std/time/struct.Duration.html#method.from_secs) is handy here.
You may also check `assert_eq!(Duration::new(5, 0), Duration::from_secs(5))`